docs/proposals/pmm/01-vm-artifacts-redesign/01-vm-artifacts-redesign.md

```diff
@@ -0,0 +1,358 @@
+# VM artifacts for PMM
```

Comment:
Great proposal. I would recommend writing it Virtual Machines specifically because it was confusing for me what VM stands for. Virtual Machines or Victoria Metrics.